Packs/SymantecBlueCoatProxySG/README.md

```diff
@@ -0,0 +1,34 @@
+# Symantec Bluecoat ProxySG
+This pack includes Cortex XSIAM content. 
+## Configuration on Server Side
+You need to configure Bluecoat ProxySG to forward Syslog messages with the access log 'main' format.
+
+Go to the ProxySG software and open the "Configuration" tab [Product Doc](https://knowledge.broadcom.com/external/article/166529/sending-access-logs-to-a-syslog-server.html);
+1. From the left sidebar, navigate to **Access Logging** > **Logs** > **Logs** tab, and create a new log with the 'main' format defined for it.
+2.  Click the **Upload Client** tab and configure your log as follows:
+
+   | Field | Value |
+   | --- | --- |
+   | Client type | Custom Client |
+   | Save the log file as | text file |
+3. In the **Custom Client Settings** dialog box, point the Custom Client to the broker, specifying the IP address and port number.
+4. Click the **Upload Schedule** tab. Make sure that the **Upload the access log** option is marked as **continuously**.
+5. From the left sidebar, navigate to **Policy** > **Visual Policy Manager**
+6. Add a new **Web Access Layer** and create a rule with the *Modify Access Logging* action. 
+7. In the **Add Access Logging Object** dialog box, enable logging for your new access log.
```

Comment:
How do you access this?

Reply:
Step 4 is from the same directory as the one mentioned for steps 1, 2, 3

Steps 6, 7 are as written in steps 5- Thank you for providing this correction.